app/editor_ui/views/prompt_views.py

```diff
@@ -212,6 +220,8 @@ class RangedPromptOptionsCreateView(
     parent_model = RangedPrompt
     parent_lookup_kwarg = "prompt_uuid"
 
+    breadcrumb = "Create a Ranged Prompt Option"
```

Comment:
I went with simple breadcrumb text for now. This can be adjusted to match preferences later.